Rakefile

```diff
@@ -0,0 +1,76 @@
+require "yaml"
+require "fileutils"
+
+desc "Validate all templates using Packer"
+task :validate do
```

Comment:
not sure you need this now that you have all your magical github action stuff

Reply:
I think it could be useful for those that use rake and want to be able to validate all .pkrvars.hcl files in 1 shot locally